docs/user-guide/kubectl/kubectl_rolling-update.md

```diff
@@ -78,6 +78,7 @@ kubectl rolling-update frontend-v1 frontend-v2 --rollback
       --dry-run[=false]: If true, print out the changes that would be made, but don't actually make them.
   -f, --filename=[]: Filename or URL to file to use to create the new replication controller.
       --image="": Image to use for upgrading the replication controller. Must be distinct from the existing image (either new image or new image tag).  Can not be used with --filename/-f
```

Comment:
I am confused. Does it still need to be distinct? I thought the point of this PR was to allow rolling-updates of the "same" (e.g. non-distinct) image so long as `--image-pull-policy=always` (as noted below).